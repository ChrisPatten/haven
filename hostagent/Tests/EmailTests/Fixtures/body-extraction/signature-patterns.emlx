745
From: sender@example.com
To: recipient@example.com
Subject: Email with Signature
Date: Thu, 21 Oct 2025 10:30:00 -0400
Message-ID: <signature@example.com>
Content-Type: text/plain; charset=utf-8

Hi there,

This is the main content of the email.

Let me know if you have any questions.

Best regards,
John Smith
Senior Developer
Acme Corporation

--
Sent from my iPhone
Please consider the environment before printing this email.

Confidentiality Notice: This email and any attachments are confidential and may be legally privileged.
