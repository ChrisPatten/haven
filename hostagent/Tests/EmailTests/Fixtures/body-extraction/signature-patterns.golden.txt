Hi there,

This is the main content of the email.

Let me know if you have any questions.

Best regards,
John Smith
Senior Developer
Acme Corporation
